addons/medical_engine/script_macros_medical.hpp

```diff
@@ -84,6 +90,19 @@
 
 #define DEFAULT_TOURNIQUET_VALUES [0,0,0,0,0,0]
 
+// Triage colors, for consistency across UIs and functions
+#define TRIAGE_COLOR_NONE      0, 0, 0, 0.9
```

Comment:
These should be defined as arrays.

Reply:
Left those out so they could be used for configs too.

Reply:
ok